samples/user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBAnimation.kt

```diff
@@ -25,7 +25,8 @@ enum class PBAnimation {
     SHARED_ELEMENT_CROSS_FRAGMENT,
     PROGRESS_API,
     TRANSITION,
-    MATERIAL_SHARED_AXIS
+    MATERIAL_SHARED_AXIS,
+    SET_CUSTOM_ANIMATIONS
```

Comment:
nit: to make better diff in the future
```suggestion
SET_CUSTOM_ANIMATIONS,
```